### DISCUSSION TITLE
ExplorerPatcher Status

### DISCUSSION THREAD

**Discussion by valinet** [valinet/ExplorerPatcher#1102] [General]
General thread containing information regarding ExplorerPatcher on various OS builds.

**Comment by LeCmnGend** [comment]
i get  crash when pressing window button, on newest pre release 22621.1413.54.5

**Reply by gracile-fr** [reply]
Same here with [KB5023778 ](https://support.microsoft.com/en-us/topic/march-28-2023-kb5023778-os-build-22621-1485-preview-d490bb51-492e-410c-871f-50ad01b0f765)(OS Build 22621.1485).

**Reply by Maksimius** [reply]
Same error after update (((

**Reply by pmatthew** [reply]
+1 unfortunately :-/

**Reply by ghost** [reply]
If you check the [known issues](https://support.microsoft.com/en-us/topic/march-28-2023-kb5023778-os-build-22621-1485-preview-d490bb51-492e-410c-871f-50ad01b0f765#knownissues) on KB5023778, it specifically mentions that it will break ExplorerPatcher and StartAllBack.

**Reply by gracile-fr** [reply]
It works now. I got a notification that the [symbols ](https://github.com/valinet/ExplorerPatcher/wiki/Symbols)have been downloaded.

**Reply by pyrates999** [reply]
You need to let symbols download.  Check for the notification indicating that.

**Reply by dooun1024** [reply]
04/13 night , I updated Windows , 04/14 (today) my windows button crash . 

Windows 11 Home
22H2
22621.1555
Windows Feature Experience Pack 1000.22640.1000.0

**Reply by Jaggu-J** [reply]
> 04/13 night , I updated Windows , 04/14 (today) my windows button crash .
> 
> Windows 11 Home 22H2 22621.1555 Windows Feature Experience Pack 1000.22640.1000.0

+1 

Same information as above, problem is after installing update KB5025239

**Reply by tonyo123O** [reply]
Same as above: windows Tuesday updated, crashed, could not get it to work, uninstalled (for now). Thank you.

**Comment by dameng324** [comment]
![QQ截图20230418090205](https://user-images.githubusercontent.com/4465571/232642405-8ca2e24e-fa2e-4e4c-b8bd-df807575994a.png)
I get this message when 4/18 morning startup.

**Reply by dameng324** [reply]
and yesterday night is normal.

**Reply by Nerexbcd** [reply]
Same Happening to me

**Reply by ManuelStaedel** [reply]
Same here

**Reply by MOTHERFUCKINGDOOMGUY** [reply]
I have that problem now for some reason it keeps crashing when i try to boot it up

**Comment by jogiicke** [comment]
Update from 9. Mai

![image](https://github.com/valinet/ExplorerPatcher/assets/133229480/a04c70dc-c243-41f0-b514-6f0f511c2d77)
![image](https://github.com/valinet/ExplorerPatcher/assets/133229480/2cb316b7-e06a-40d3-b82d-2047171ef740)

**Reply by mohawk1990** [reply]
I am getting this same notice. I would appreciate some guidance for a workaround as an update as noted in the status popup is NOT available.  For reference, Here are my version specs:


Edition	Windows 11 Pro Insider Preview
Version	26H1
Installed on	‎19-‎Jun-‎26
Evaluation expires on	‎11-‎Aug-‎26 14:09 PM
OS build	28120.2374
Experience	Windows Feature Experience Pack 1000.26100.398.0

**Reply by pyrates999** [reply]
You responded to a comment from 2 years ago, your issue is not the same.

**Comment by pyrates999** [comment]
If that happens, uninstall explorer patcher.  Check the wiki for how to do that if you don't know how.

**Reply by jstucken** [reply]
I'm on Windows 11 version 22H2 and am experiencing the same error like @jogiicke. My windows updated very recently about 10 minutes beforehand, so suspect linked to recent update. I had to uninstall ExplorerPatcher for now.

Keenly waiting a fix. If I can help test things in any way I'm happy to do so. This is such an excellent program, I rely on it a lot. Thank you to the developers for what you do :)

**Reply by pyrates999** [reply]
Check your windows version and build number to be sure you didn't install a preview update.  Those are not supported because Microsoft doesn't provide symbols to download for them.

**Comment by jstucken** [comment]
Deleting the following registry key as per [this thread](https://github.com/valinet/ExplorerPatcher/issues/1434#issuecomment-1543972982) then reinstalling fixed the issue for me :)

`Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher`

I lost some of my prior configuration settings as expected, but no big deal, easily fixed.

**Reply by edvyan** [reply]
Thanks, that really works!

**Reply by swalsh76** [reply]
> This also worked for me!

**Comment by pyrates999** [comment]
Works on Windows 10 22H2 build 19045.2965
explorer patcher version: 22621.1555.55.1

**Comment by Msprg** [comment]
ExplorerPatcher 22621.1992.56.3 running on Windows 11 build 10.0.22621.2134 With downloaded symbols seems to be working correctly 🎉 , even with Windows 10 style taskbar and `Do not change the taskbar context menu` setting disabled.

Windows updates left untouched.

**Comment by dstewartwhite** [comment]
File Explorer crashing Win 11 22621.2215 ExplorerPatcher [Version: 22621.1992.56.3]. Worked Ok until I updated my Nvidia Driver to get another programme working

**Reply by pyrates999** [reply]
That is a preview update: https://support.microsoft.com/en-us/topic/august-22-2023-kb5029351-os-build-22621-2215-preview-9af25662-083a-43f5-b3a7-975fe25cc692

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Comment by dstewartwhite** [comment]
Followed the above and instead of getting the ExplorerPatcher Crash dialog box. Windows returned a Critical Error message that stated that the Start Menu was unavailable and Widows would try and fix it at next Logon, then forced a logoff.
Maintenance Log shows:

Description
Faulting Application Path:	C:\Windows\explorer.exe

Problem signature
Problem Event Name:	APPCRASH
Application Name:	explorer.exe
Application Version:	10.0.22621.2070
Application Timestamp:	2bd7252c
Fault Module Name:	dxgi.dll
Fault Module Version:	22621.1992.56.3
Fault Module Timestamp:	64e6228c
Exception Code:	c0000005
Exception Offset:	000000000001099d
OS Version:	10.0.22621.2.0.0.256.48
Locale ID:	2057
Additional Information 1:	b1ca
Additional Information 2:	b1ca616edac811e83209cf424c3a402d
Additional Information 3:	3874
Additional Information 4:	3874afad7f3afac1777789c86289696f

Extra information about the problem
Bucket ID:	d3c9ead718e98f47f8f4470497c11c8e (1798140236279127182)

**Reply by pyrates999** [reply]
that is also a preview update: https://support.microsoft.com/en-gb/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

**Comment by dstewartwhite** [comment]
My build does not list KB5028254 as installed, but it does for 2023-08 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5029351) but this one is not listed in the Uninstall Updates list. So do not know how to remove it

**Reply by pyrates999** [reply]
here's how to uninstall it: https://4it.com.au/kb/article/how-to-uninstall-windows-updates-using-the-command-prompt/

**Comment by dstewartwhite** [comment]
Tried the link and it appeared to work, no error messages, but it is still listed in the Update History so I assume it has not uninstalled. I will just have to wait until the next Stable build
Many thanks for all your help

**Reply by pyrates999** [reply]
Try running winver.  It will show you the windows version and build so you can then check if the update was uninstalled.

The history of installing windows updates in the settings app is just another log and can be cleared while the update still is technically installed.  You can only be sure what windows updates were installed by going into the control panel.  From there the only time an update will not be shown is when you have uninstalled it.

**Comment by Alex423243** [comment]
Installing a new windows 11 update (24H2, OS Build 26100.4202) causes Explorer to start repeatedly crash. I switched to the windows 11 taskbar and it worked fine so i switched back to the Windows 10 (Explorer Patcher) Taskbar then explorer started to crash again. Does anybody know a fix?

**Reply by pyrates999** [reply]
check if you enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Comment by foxd01** [comment]
<img width="1823" height="789" alt="image" src="https://github.com/user-attachments/assets/3f6dd308-94a2-4a9f-b277-eb2fb6d06f63" />

**Comment by foxd01** [comment]
The latest version installer on windows 12 has the bugs above.
After the "5 crashes" message,  the uninstaller seems not to work, it leaves the boot process in place which generates the exact same death loop.

**Comment by EJH-BAE** [comment]
> General thread containing information regarding ExplorerPatcher on various OS builds.

I crashed on explorerpatcher pre-release
<img width="496" height="147" alt="image" src="https://github.com/user-attachments/assets/5c7e6c48-0d11-4342-95f2-0912b8a86803" />
Windows ver : 25H2

**Reply by pyrates999** [reply]
see here: https://github.com/valinet/ExplorerPatcher/issues/4957
